src/qt/coincontroldialog.cpp

```diff
@@ -340,9 +335,10 @@ void CoinControlDialog::radioListMode(bool checked)
 // checkbox clicked by user
 void CoinControlDialog::viewItemChanged(QTreeWidgetItem* item, int column)
 {
-    if (column == COLUMN_CHECKBOX && item->data(COLUMN_ADDRESS, TxHashRole).toString().length() == 64) // transaction hash is 64 characters (this means it is a child node, so it is not a parent node in tree mode)
-    {
-        COutPoint outpt(TxidFromString(item->data(COLUMN_ADDRESS, TxHashRole).toString().toStdString()), item->data(COLUMN_ADDRESS, VOutRole).toUInt());
+    if (column != COLUMN_CHECKBOX) return;
+    auto txid{Txid::FromHex(item->data(COLUMN_ADDRESS, TxHashRole).toString().toStdString())};
+    if (txid) { // a valid txid means this is a child node, and not a parent node in tree mode
```

Comment:
Just a comment: Still not sure what ends up being more readable. This, or:
```
if (auto txid{...}; txid) {
...
```

Reply:
Usually I'd have preferred the `if (auto txid{...}) {` approach (no need for the `; txid` I think), but since we have the docstring anyway I thought the current approach was slightly more readable.

Reply:
Though of the same, both are fine with me as well